web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js

```diff
@@ -70,6 +70,7 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
             var $table = $('<table>').addClass(
                 'o_list_view table table-condensed table-striped'
             );
+            $table.addClass('o_x2many_2d_matrix')
```

Comment:
Why not put this class in the addClass on the previous line?

Reply:
Frankly speaking, I don't know. 80-line formatting I guess. How to better format that?

Reply:
joining 2 strings is a solution.

Reply:
```
            var $table = $('<table>').addClass(
                'o_list_view table table-condensed table-striped' +
                'o_x2many_2d_matrix'
            );
```
This is fine (for me though)

Reply:
Done